tests/e2e/operator/operator_test.go

```diff
@@ -310,7 +312,7 @@ spec:
 		Success("Operator uninstalled")
 
 		By("Deleting the CRDs")
-		Expect(kubectl.DeleteCRDs(crds)).To(Succeed(), "CRDs failed to be deleted")
+		Expect(kubectl.DeleteCRDs(expectedCRDList)).To(Succeed(), "CRDs failed to be deleted")
```

Comment:
This line looks strange, because of the `expectedCRDList` name. Maybe rename to `istioCRDs` or `sailCRDs`?